README.md

```diff
@@ -59,4 +59,4 @@ cAdvisor aims to improve the resource usage and performance characteristics of r
 
 ## Community
 
-Contributions, questions, and comments are all welcomed and encouraged! cAdvisor developers hang out on [Freenode](http://freenode.net/) IRC in the [#google-containers](http://webchat.freenode.net/?channels=google-containers) room & [Slack](https://kubernetes.slack.com) (get an invitation [here](http://slack.kubernetes.io/)). We also have the [google-containers Google Groups mailing list](https://groups.google.com/forum/#!forum/google-containers).
+Contributions, questions, and comments are all welcomed and encouraged! cAdvisor developers hang out on [Freenode](http://freenode.net/) IRC in the [#google-containers](http://webchat.freenode.net/?channels=google-containers) room and [Slack](https://kubernetes.slack.com) #sig-autoscaling channel (get an invitation [here](http://slack.kubernetes.io/)). We also have the [google-containers Google Groups mailing list](https://groups.google.com/forum/#!forum/google-containers).
```

Comment:
I respond to questions there whenever I can. We can remove it though.

On Tue, Jun 7, 2016 at 2:58 PM, Tim St. Clair <notifications@github.com>
wrote:

> In README.md
> <https://github.com/google/cadvisor/pull/1323#discussion_r66160638>:
>
> > @@ -59,4 +59,4 @@ cAdvisor aims to improve the resource usage and performance characteristics of r
> >
> >  ## Community
> >
> > -Contributions, questions, and comments are all welcomed and encouraged! cAdvisor developers hang out on [Freenode](http://freenode.net/) IRC in the [#google-containers](http://webchat.freenode.net/?channels=google-containers) room & [Slack](https://kubernetes.slack.com) (get an invitation [here](http://slack.kubernetes.io/)). We also have the [google-containers Google Groups mailing list](https://groups.google.com/forum/#!forum/google-containers).
> > +Contributions, questions, and comments are all welcomed and encouraged! cAdvisor developers hang out on [Freenode](http://freenode.net/) IRC in the [#google-containers](http://webchat.freenode.net/?channels=google-containers) room and [Slack](https://kubernetes.slack.com) #sig-autoscaling channel (get an invitation [here](http://slack.kubernetes.io/)). We also have the [google-containers Google Groups mailing list](https://groups.google.com/forum/#!forum/google-containers).
>
> Is the freenode IRC line still accurate?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/google/cadvisor/pull/1323/files/936d1db3f500c1b73648df84afc52e7091877bd0#r66160638>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGvIKEpcnYhsgYHqGPd1-sTkBB4vjjB1ks5qJel-gaJpZM4IwVsw>
> .
>